PROCESSORS.md

```diff
@@ -2875,6 +2877,41 @@ In the list below, the names of required properties appear in bold. Any other pr
 
 
+## SplitRecord
+
+### Description
+
+Splits up an input FlowFile that is in a record-oriented data format into multiple smaller FlowFiles
+
+### Properties
+
+In the list below, the names of required properties appear in bold. Any other properties (not in bold) are considered optional. The table also indicates any default values, and whether a property supports the NiFi Expression Language.
+
+| Name                  | Default Value | Allowable Values | Description                                                                                                                   |
+|-----------------------|---------------|------------------|-------------------------------------------------------------------------------------------------------------------------------|
+| **Record Reader**     |               |                  | Specifies the Controller Service to use for reading incoming data                                                             |
+| **Record Writer**     |               |                  | Specifies the Controller Service to use for writing out the records                                                           |
+| **Records Per Split** |               |                  | Specifies how many records should be written to each 'split' or 'segment' FlowFile<br/>**Supports Expression Language: true** |
+
+### Relationships
+
+| Name     | Description                                                                                                                                                       |
+|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| failure  | If a FlowFile cannot be transformed from the configured input format to the configured output format, the unchanged FlowFile will be routed to this relationship. |
+| original | Upon successfully splitting an input FlowFile, the original FlowFile will be sent to this relationship.                                                           |
+| splits   | The individual 'segments' of the original FlowFile will be routed to this relationship.                                                                           |
+
+### Output Attributes
+
+| Attribute                 | Relationship | Description                                                                                                                    |
+|---------------------------|--------------|--------------------------------------------------------------------------------------------------------------------------------|
+| record.count              |              | The number of records in the FlowFile. This is added to FlowFiles that are routed to the 'splits' Relationship.                |
+| fragment.identifier       |              | All split FlowFiles produced from the same parent FlowFile will have the same randomly generated UUID added for this attribute |
```

Comment:
We changed this to the ID of the parent flow file for `SegmentContent`; I think we should change it in the same way here for the same reason, and also to be consistent with `SegmentContent`.